build/toolchain/linux/BUILD.gn

```diff
@@ -114,6 +114,16 @@ clang_toolchain("clang_x64") {
   }
 }
 
+clang_toolchain("cobalt_clang_x64") {
```

Comment:
not the best idea as this will lead to multiple cobalt toolchains for each architecture.

in cobalt code whenever i write if(current_toolchain == cobalt_toolchain) : i dont want it to apply for host_toolchain as it does currently MAybe there's a better way of doing this ? .

Reply:
maybe a better way is to change  the label for the linux host toolchain to something like linux_host_clang_x64 ?

Reply:
Idealy the host toolchain would not use our customizations for libc and other modular specific flags we need.  The issue was forced this change was an error compiling nasm, adding the musl include paths caused a weird error: https://buganizer.corp.google.com/issues/371241293#comment19 .

Reply:
Isn't the nasm tool explicitly configured to be built with the host toolchain?
Maybe some of the hermetic/musl flags just need to be applied only if the toolchain is not the host toolchain?

Reply:
correct, that's one of the reasons i need the separation. You actually get linker errors when you try to link with musl - and i don't think we want to do that anyways.  Like we should try and use the host toolchain from chromium as it is as much as possible.

Reply:
Maybe check is_starboard? I would expect that is would not be set for the host_toolchain.

Reply:
is_starboard is currently a global variable and will get applied to any toolchain

Reply:
+1 ^ . To maybe answer yavor's question more , i don' think i can get away with some gn variable. the separation is  via the toolchain label.